.github/scripts/ci/build-pr-comment.sh

```
@@ -0,0 +1,232 @@
+ELAPSED=$(( NOW - JOB_START ))
+DURATION_STR="$(( ELAPSED / 60 ))m $(( ELAPSED % 60 ))s"
+
+# Extract tc_total from embedded RESOLVED_JSON block; resolve feature breakdown dynamically
+TC_TOTAL=0
+if [[ -n "$REPORT_FILE" && -f "$REPORT_FILE" ]]; then
+  RESOLVED_JSON=$(sed -n '/++RESOLVED_JSON_START++/,/++RESOLVED_JSON_END++/p' "$REPORT_FILE" | sed '1d;$d')
+  TC_TOTAL=$(echo "$RESOLVED_JSON" | jq -r '.tc_total // 0')
+fi
+
+FILTER_ARGS_RAW=$(echo "$RESOLVED_JSON" | jq -r '.filter_args // [] | join(" ")' 2>/dev/null || echo '')
+QA_JSON=$(node packages/rangelink-vscode-extension/scripts/resolve-qa-labels.js $FILTER_ARGS_RAW --json 2>/dev/null || echo '{"groups":[]}')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)